File: CayStudyModule1/test_task6/Customer.js
let Customer = function () {

    this.setNameCustomer = function (nameCustomer) {
        this.nameCustomer = nameCustomer;
    };
    this.getNameCustomer = function () {
        return this.nameCustomer;
    };
    //name
    this.setIdCard = function (IdCard) {
        this.IdCard = IdCard;
    };
    this.getIdCard = function () {
        return this.IdCard;
    };
    //IdCard
    this.setBirthday = function (Birthday) {
        this.Birthday = Birthday;
    };
    this.getBirthday = function () {
        return this.Birthday;
    };
    //birthday
    this.setEmail = function (Email) {
        this.Email = Email;
    };
    this.getEmail = function () {
        return this.Email;
    };
    //Email
    this.setAddress = function (Address) {
        this.Address = Address;
    };
    this.getAddress = function () {
        return this.Address;
    };
    //Address
    this.setTypeCustomer = function (TypeCustomer) {
        this.TypeCustomer = TypeCustomer;
    };
    this.getTypeCustomer = function () {
        return this.TypeCustomer;
    };
    //Type Customer
    this.setDiscount = function (Discount) {
        this.Discount = Discount;
    };
    this.getDiscount = function () {
        return this.Discount;
    };
    //Discount
    this.setAccompanying = function (Accompanying) {
        this.Accompanying =Accompanying;
    };
    this.getAccompanying = function () {
        return this.Accompanying;
    };
    //Accompanying
    this.setTypeRoom = function (TypeRoom) {
        this.TypeRoom = TypeRoom;
    };
    this.getTypeRoom = function () {
        return this.TypeRoom;
    };
    //Type Room
    this.setRentDays = function (RentDays) {
        this.RentDays = RentDays;
    };
    this.getRentDays = function () {
        return this.RentDays;
    };
    //Rent Days
    this.setTypeService = function (TypeService) {
        this.TypeService = TypeService;
    };
    this.getTypeService = function () {
        return this.TypeService;
    };
    //Type Service
    this.totalPays=function(){
        let moneyPerDays = 0;
        if (this.getTypeService() === "villa") {
            moneyPerDays=500;
        }
        else  if (this.getTypeService() === "house") {
            moneyPerDays=300;
        }
        else if(this.getTypeService()==="room")  {
            moneyPerDays=100;
        }
        return moneyPerDays*parseFloat(this.getRentDays())*(1-parseFloat(this.getDiscount()))
    }
};

File: Module2/src/CaseStudy2/Models/Customer.java
package CaseStudy2.Models;

public class Customer {
    private String Name;
    private String Date;
    private String gender;
    private String passPort;
    private String phoneNumber;
    private String Email;
    private String typeCustomer;
    private String address;
    private String typeService;

    public Customer(String name, String date, String gender, String passPort, String phoneNumber, String email, String typeCustomer, String address, String typeService) {
        Name = name;
        Date = date;
        this.gender = gender;
        this.passPort = passPort;
        this.phoneNumber = phoneNumber;
        Email = email;
        this.typeCustomer = typeCustomer;
        this.address = address;
        this.typeService = typeService;
    }
    public Customer() {

    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    public String getDate() {
        return Date;
    }

    public void setDate(String date) {
        Date = date;
    }

    public String isGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public String getPassPort() {
        return passPort;
    }

    public void setPassPort(String passPort) {
        this.passPort = passPort;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public String getEmail() {
        return Email;
    }

    public void setEmail(String email) {
        Email = email;
    }

    public String getTypeCustomer() {
        return typeCustomer;
    }

    public void setTypeCustomer(String typeCustomer) {
        this.typeCustomer = typeCustomer;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getTypeService() {
        return typeService;
    }


    public String showInformation() {
        return "Customer{" +
                "Name='" + Name + '\'' +
                ", Date='" + Date + '\'' +
                ", gender=" + gender +
                ", passPort='" + passPort + '\'' +
                ", phoneNumber='" + phoneNumber + '\'' +
                ", Email='" + Email + '\'' +
                ", typeCustomer='" + typeCustomer + '\'' +
                ", address='" + address + '\'' +
                ", typeService=" + typeService +
                '}';
    }

    public void setTypeService(String typeService) {
        this.typeService = typeService;
    }



}
